identity-api/src/main/java/life/qbic/identity/api/UserInfo.java

```diff
@@ -10,6 +11,6 @@
  * @since 1.0.0
  */
 public record UserInfo(String id, String fullName, String emailAddress, String platformUserName,
-                       boolean isActive) implements Serializable {
+                       boolean isActive, Optional<String> oidcId) implements Serializable {
```

Comment:
You should not use Optional in a record. Optional is not serializable and may break the record. Use `null` in a record instead.

Reply:
Good point, I adjusted the record to contain a string and check for null values explictly